### ISSUE TITLE
1.16 is not working for me

### ISSUE THREAD

**Issue by franiscoder** [way2muchnoise/CurseForgeBadge#26]
1.16 is not appearing on the widget for versions for me.
[![](http://cf.way2muchnoise.eu/versions/mo-structures.svg)](https://www.curseforge.com/minecraft/mc-mods/mo-structures) 
is how i do it on the readme, here is the project and readme link: 
https://github.com/franiscoder/mostructures/blob/1.16.x/README.md

**Comment by way2muchnoise** [comment]
I use some undocumented API that the Twitch client uses to pull all the info.
So it could either be the fact that it is Fabric (don't think they are on the twitch client).
Or the Twitch client not yet supporting 1.16.

I used to scrape the website, but got banned once the put up CloudFlare protection.
I hope Overwolf provides a proper API once they are in control of CurseForge (an API is on their roadmap).

TL;DR: Can't fix this myself, it all relies on the available data.
